lib/manager/gradle-wrapper/__fixtures__/gradle-wrapper-all.properties

```diff
@@ -2,6 +2,6 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-# @See https://gradle.org/releases/
+
+# distributionUrl includes distribution type "bin"
 distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.3-all.zip
-distributionSha256Sum=336b6898b491f6334502d8074a6b8c2d73ed83b92123106bd4bf837f04111043
```

Comment:
Why do you removed the hash? This is an `all` url, so your comment seems wrong.
I also see no rfeason to test  `all` and `bin`  distribution seperatly

Reply:
`all` and `bin` have been tested seperately before, I did not add this test case.
However, I think it's useful to test both cases, as these are the supported ones and explicitly defined in the regex used for the version extraction.

I'll re-add the hashsum, even though they do not matter to the test case, as far as I know.